docs/framework/guides/tutorials/implementing-a-block-widget.md

```diff
@@ -846,7 +846,7 @@ Change one more thing before you move forward &mdash; disallow `simpleBox` insid
 ```js
 // simplebox/simpleboxediting.js
 
-// ... imports
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.